src/translations/hindi/README.md

```diff
@@ -29,165 +25,179 @@ lang: hi
+* 'होस्ट ऑब्जेक्ट्स' और 'नेटिव ऑब्जेक्ट्स' के बीच क्या अंतर है?
+* 'function Person() {}', 'var person = Person()', और 'var person = new Person()' के उपयोग के बारे में विवाद क्या है?
+* 'function foo() {}' और 'var foo = function() {}' के बीच 'foo' के उपयोग के विभिन्नताओं का विवरण करें
+* 'Function.call' और 'Function.apply' क्या काम करते हैं? इन दोनों के बीच क्या महत्वपूर्ण अंतर है?
+* 'Function.prototype.bind' का विवरण करें।
+* फीचर डिटेक्शन (feature detection), फीचर इनफरेंस (feature inference), और UA स्ट्रिंग का उपयोग करने के बीच क्या अंतर है?
+* 'होइस्टिंग' का विवरण करें।
+* टाइप कोएर्शन क्या है? JavaScript कोड में टाइप कोएर्शन पर भरपूर निर्भरता की सामान्य चूकें क्या होती हैं?
+* घटना बबलिंग (event bubbling) को समझाएं।
+* घटना कैप्चरिंग (event capturing) को समझाएं।
+* 'एट्रिब्यूट' और 'प्रॉपर्टी' के बीच क्या अंतर है?
+* बिल्ट-इन JavaScript ऑब्जेक्ट्स को विस्तारित करने के फायदे और नुकसान क्या हैं?
+* '==' और '===' के बीच क्या अंतर है?
+* JavaScript के संदर्भ में सेम-आरिजिन पॉलिसी (same-origin policy) को समझाएं।
+* इसे 'टर्नरी ऑपरेटर' क्यों कहा जाता है, 'टर्नरी' शब्द का क्या मतलब है?
+* स्ट्रिक्ट मोड क्या है? इसके प्रयोग करने के कुछ फायदे/हानियां क्या हैं?
+* JavaScript कोड को JavaScript में कंपाइल करने वाली भाषा में लिखने के कुछ फायदे/हानियां क्या हैं?
+* JavaScript कोड को डिबग करने के लिए आप कौन-कौन से उपकरण और तकनीकों का उपयोग करते हैं?
+* परिवर्तनशील और अपरिवर्तनशील ऑब्जेक्ट्स के बीच क्या अंतर है?
+* JavaScript में एक अपरिवर्तनशील ऑब्जेक्ट का एक उदाहरण क्या है?
+* अपरिवर्तनशीलता के फायदे और नुकसान क्या हैं?
+* आप अपने कोड में अपरिवर्तनशीलता कैसे प्राप्त कर सकते हैं?
+* सिंक्रोनस और एसिंक्रोनस फ़ंक्शन्स के बीच क्या अंतर है?
+* घटना लूप (event loop) क्या है?
+* कॉल स्टैक और टास्क क्यू के बीच क्या अंतर है?
+* 'let', 'var', या 'const' का उपयोग करके बनाई गई चरों (variables) के बीच क्या अंतर है?
+* ES6 क्लास और ES5 फ़ंक्शन कंस्ट्रक्टर्स के बीच क्या अंतर है?
+* नए ऐरो => फ़ंक्शन सिंटैक्स के एक उपयोग केस प्रदान कर सकते हैं? इस नए सिंटैक्स का विभिन्नता अन्य फ़ंक्शन्स से क्या है?
+* एक कंस्ट्रक्टर में एक मेथड के लिए ऐरो सिंटैक्स का उपयोग करने के क्या फायदे हैं?
+* उच्च-क्रम फ़ंक्शन (higher-order function) की परिभाषा क्या है?
+* कृपया एक ऑब्जेक्ट या एक एरे को डिस्ट्रक्चर (destructuring) करने का एक उदाहरण दे सकते हैं?
+* क्या आप ES6 टेम्पलेट लिटरल्स का उपयोग करके एक स्ट्रिंग उत्पन्न करने का एक उदाहरण दे सकते हैं?
+* क्या आप करी फ़ंक्शन का एक उदाहरण और इस सिंटैक्स के फ़ायदे के बारे में बता सकते हैं?
+* 'spread सिंटैक्स' का उपयोग करने के फ़ायदे क्या हैं और यह 'rest सिंटैक्स' से कैसे अलग है?
+* फ़ाइलों के बीच कोड साझा कैसे किया जा सकता है?
+* आपको स्टेटिक क्लास सदस्य (static class members) बनाने की क्या आवश्यकता हो सकती है?
+* 'while' और 'do-while' लूप्स के बीच क्या अंतर है?
+* एक प्रॉमिस (promise) क्या होता है? कहाँ और कैसे आप प्रॉमिस का उपयोग करेंगे?
+* JavaScript के साथ कोडिंग करते समय ऑब्जेक्ट ओरिएंटेड प्रोग्रामिंग (Object Oriented Programming) के सिद्धांतों का उपयोग करने की बात करें।
+
+## कोडिंग प्रश्न
```

Comment:
#### [[ ⬆ ]] ( #toc ) < a name = ' code-questions' >कोडिंग प्रश्न</ a >

Reply:
Updated this as well